test/bigtest/tests/line-edit-test.js

```diff
@@ -195,4 +195,29 @@ describe('Line edit test', () => {
       });
     });
   });
+
+  describe('Check existing Item Details Title positive case', () => {
+    const EMPTY_STRING = '';
+
+    beforeEach(async function () {
+      await lineEditPage.itemDetailsAccordion.inputTitle('TEST_VALUE');
+      await lineEditPage.updateLineButton.click();
+    });
+
+    it('Shouldn\'t provide any warning message', () => {
+      expect(lineEditPage.itemDetailsAccordion.errorTitle).to.be.equal(EMPTY_STRING);
+    });
+  });
+
+  describe('Check existing Item Details Title negative case', () => {
```

Comment:
nit: sounds not good to me `'Check existing Should provide title warning`

Reply:
will fix next commit